zerver/webhooks/newrelic/doc.md

```diff
@@ -1,27 +1,121 @@
+# New Relic ClickUp integration
+
 New Relic can send messages to a Zulip channel for incidents.
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-1. On [New Relic](https://one.newrelic.com),
-  select **Alerts & AI**.
+1. Go to the [**Destinations**][destinations] menu and select **Webhook** in
+   the **Add a destination** section. ([*Official Guide*][official-guide])
 
-1. Navigate to **Notification channels**.
+1. Set **Endpoint URL** to the URL you generated.
 
-1. Create a new notification channel. Select channel type of **Webhook**, choose a name (e.g., "Zulip"), enter the webhook url created earlier as **Base Url**.
+1. Go to the [**Workflows**][workflows] menu and click on **+ Add a Workflow**
+   button. After filtering the trigger condition, select on the **Webhook**
+   option in the **Add Channel** section. ([*Workflow doc*][workflow-doc])
 
-1. It should look like:
-  ![](/static/images/integrations/newrelic/newrelic.png)
+1. Once you're in the **Edit notification message** menu, select the
+   destination for Zulip you've just created earlier.
 
-1. The webhook works with the default payload, click **Create channel**.
+1. If you want to include additional information to the notification, refer
+   to this [section](#displaying-custom-fields). There's also an
+   optional [base payload template](#recommended-base-payload-template) we
+   recommend you to use.
 
-1. After creating the channel send a test notification to make sure it works.
+1. Select **Save message** and the **Activate Workflow** to finish.
+
+{end_tabs}
 
 {!congrats.md!}
 
 ![](/static/images/integrations/newrelic/001.png)
+
 ![](/static/images/integrations/newrelic/002.png)
```

Comment:
Hmm, I tried capturing it without the "selected" outline but couldn't manage to do it without adding more messages following it. However, with this approach, the screenshot feels a bit awkward because the bottom looks cut off. 
![image](https://github.com/zulip/zulip/assets/108744694/5e05a5d2-d5c3-4fe0-ab59-6d36781126e6)
I'll leave it with only 1 screenshot for now..

Reply:
Can't use the `generate-integration-docs-screenshot` because it will generate a closed notification.